docs/md_book/src/intro.md

```diff
@@ -15,9 +15,9 @@ and its source code is completely yours to tinker with. The code is available on
 [am]: https://github.com/amethyst/laminar/tree/master
 
 ## Motivation
-Laminar is fully written in Rust and therefore it has no garbage collector, no data-races, and memory safety. 
+Laminar is fully written in Rust and therefore has no garbage collector, no data-races, and is completely memory safety.
```

Comment:
`completely memory safety.`

Reply:
I'm also not entirely sure of the "is completely memory safe". It isn't exactly true.

Reply:
@jstnlef Rust without `unsafe` is completely memory safe. We should disallow unsafe code via the crate lint before asserting this as a fact about laminar.